### PR TITLE
Add left over rails packages

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

File: packages/foreman/rubygem-nokogiri/rubygem-nokogiri.spec [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+%exclude %{gem_instdir}/.gemtest
+%exclude %{gem_instdir}/.travis.yml
+%license %{gem_instdir}/LICENSE-DEPENDENCIES.md
+%license %{gem_instdir}/LICENSE.md
+%{gem_instdir}/Manifest.txt
+%{gem_instdir}/ROADMAP.md
+%{gem_instdir}/SECURITY.md
+%{gem_instdir}/STANDARD_RESPONSES.md
+%{gem_instdir}/Y_U_NO_GEMSPEC.md
+%{gem_instdir}/appveyor.yml
```
Comment (ekohl) [inline packages/foreman/rubygem-nokogiri/rubygem-nokogiri.spec]: This feels like something to exclude

File: packages/foreman/rubygem-nokogiri/rubygem-nokogiri.spec [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+%{gem_extdir_mri}
+%{gem_instdir}/.autotest
+%{gem_instdir}/.cross_rubies
+%{gem_instdir}/.editorconfig
+%exclude %{gem_instdir}/.gemtest
+%exclude %{gem_instdir}/.travis.yml
+%license %{gem_instdir}/LICENSE-DEPENDENCIES.md
+%license %{gem_instdir}/LICENSE.md
+%{gem_instdir}/Manifest.txt
+%{gem_instdir}/ROADMAP.md
```
Comment (ekohl) [inline packages/foreman/rubygem-nokogiri/rubygem-nokogiri.spec]: This and a few others belong in the `%doc` package IMHO.

File: packages/foreman/rubygem-nokogiri/rubygem-nokogiri.spec [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
+
+%files
+%dir %{gem_instdir}
+%{_bindir}/nokogiri
+%exclude %{gem_instdir}/ext
+%{gem_extdir_mri}
+%{gem_instdir}/.autotest
+%{gem_instdir}/.cross_rubies
+%{gem_instdir}/.editorconfig
```
Comment (ekohl) [inline packages/foreman/rubygem-nokogiri/rubygem-nokogiri.spec]: Probably safe to exclude. Probably true for this and the 3 hidden files above

**Comment by zjhuntin** [comment]
@ekohl updated and made the changes suggested, see whatcha think!

**Comment by zjhuntin** [comment]
Also, should only be merged after https://github.com/theforeman/foreman-packaging/pull/4578

**Pull request merged**